universal-application-tool-0.0.1/test/services/applicant/question/ApplicantQuestionTest.java

```diff
@@ -2,114 +2,35 @@
-          ImmutableMap.of(Locale.US, "help text"),
-          ImmutableListMultimap.of(Locale.US, "option 1", Locale.US, "option 2"));
-  private static final DropdownQuestionDefinition dropdownQuestionDefinition =
-      new DropdownQuestionDefinition(
-          1L,
-          "question name",
-          Path.create("applicant.my.path.name"),
-          Optional.empty(),
-          "description",
-          LifecycleStage.ACTIVE,
-          ImmutableMap.of(Locale.US, "question?"),
-          ImmutableMap.of(Locale.US, "help text"),
-          ImmutableListMultimap.of(
-              Locale.US,
-              "option 1",
-              Locale.US,
-              "option 2",
-              Locale.FRANCE,
-              "un",
-              Locale.FRANCE,
-              "deux"));
-  private static final TextQuestionDefinition textQuestionDefinition =
-      new TextQuestionDefinition(
-          1L,
-          "question name",
-          Path.create("applicant.my.path.name"),
-          Optional.empty(),
-          "description",
-          LifecycleStage.ACTIVE,
-          ImmutableMap.of(Locale.US, "question?"),
-          ImmutableMap.of(Locale.US, "help text"));
-  private static final NameQuestionDefinition nameQuestionDefinition =
-      new NameQuestionDefinition(
-          1L,
-          "question name",
-          Path.create("applicant.my.path.name"),
-          Optional.empty(),
-          "description",
-          LifecycleStage.ACTIVE,
-          ImmutableMap.of(Locale.US, "question?"),
-          ImmutableMap.of(Locale.US, "help text"));
-  private static final NumberQuestionDefinition numberQuestionDefinition =
-      new NumberQuestionDefinition(
-          1L,
-          "question name",
-          Path.create("applicant.my.path.name"),
-          Optional.empty(),
-          "description",
-          LifecycleStage.ACTIVE,
-          ImmutableMap.of(Locale.US, "question?"),
-          ImmutableMap.of(Locale.US, "help text"));
-  private static final AddressQuestionDefinition addressQuestionDefinition =
-      new AddressQuestionDefinition(
-          1L,
-          "question name",
-          Path.create("applicant.my.path.name"),
-          Optional.empty(),
-          "description",
-          LifecycleStage.ACTIVE,
-          ImmutableMap.of(Locale.US, "question?"),
-          ImmutableMap.of(Locale.US, "help text"));
-  private static final RadioButtonQuestionDefinition radioButtonQuestionDefinition =
-      new RadioButtonQuestionDefinition(
-          1L,
-          "question name",
-          Path.create("applicant.my.path.name"),
-          Optional.empty(),
-          "description",
-          LifecycleStage.ACTIVE,
-          ImmutableMap.of(Locale.US, "question?"),
-          ImmutableMap.of(Locale.US, "help text"),
-          ImmutableListMultimap.of(Locale.US, "option 1", Locale.US, "option 2"));
+  private static final QuestionDefinition checkboxQuestionDefinition =
```

Comment:
(non-blocking) these don't really need to be `static final`s since the `TestQuestionBank` keeps questions cached. These could be called by the test methods directly, instead of referencing a static final variable.

Reply:
I'm going to separate this file out into a different PR because it is causing me issues when run as with cross-test-cases.  However, I will make sure the static final is removed in the next PR for `ApplicantQuestionTest`